src/ntfs/write.rs

```diff
@@ -224,181 +200,283 @@ struct Fares<'a> {
     ticket_use_restrictions: &'a Collection<TicketUseRestriction>,
 }
 
-fn has_constraints(ticket_use: &TicketUse) -> bool {
-    ticket_use
-        .max_transfers
-        .filter(|&mt| mt != 0)
-        .or(ticket_use.boarding_time_limit)
-        .or(ticket_use.alighting_time_limit)
-        .is_some()
-}
-
-fn get_prices<'a>(
-    ticket_prices: &'a Collection<TicketPrice>,
-    ticket_id: &str,
-) -> Vec<&'a TicketPrice> {
-    ticket_prices
+// Returns Ok(()) if each ticket_use_id appears at most once in fares
+// returns an error otherwise
+fn check_uniqueness_of_ticket_use_ids(fares: &Fares) -> Result<()> {
```

Comment:
Uniqueness is guaranteed on ticket_uses (it's a CollectionWithId). 
I hardly see the relevance of this function. Did you have the case?

Reply:
you're right ! I removed the function